daemon/config/config.go

```diff
@@ -127,6 +127,10 @@ type Config struct {
 
 	// CgroupDriver sets cgroup driver for all containers
 	CgroupDriver string `json:"cgroup-driver,omitempty"`
+
+	// InsecureRegistries sets insecure registries to allow to pull
+	// insecure registries.
+	InsecureRegistries []string `json:"insecure-registries,omitempty"`
```

Comment:
add daemon test for this config?

Reply:
it is hard to add this case right now because we need to download the registry and push the image in there. I am not sure how to make it easier to add it into repo.